packages/grid/helm/syft/templates/registry-statefulset.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: registry
+  labels:
+    app.kubernetes.io/name: {{ .Chart.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: registry
+    app.kubernetes.io/managed-by: Helm
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ .Chart.Name }}
+      app.kubernetes.io/component: registry
+      app.kubernetes.io/managed-by: Helm
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ .Chart.Name }}
+        app.kubernetes.io/component: registry
+        app.kubernetes.io/managed-by: Helm
+    spec:
+      containers:
+      - image: registry:2
```

Comment:
NITPICK: we could give the full identifier `docker.io/library/registry:2`
as with other container runtimes, the default registry assumption might be different.

Reply:
not required, defaults to docker anyway.